lib/stats.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { Stats, convertMSToMicroseconds } = require('@tracerbench/stats');
+function sevenFigureSummary(summary) {
```

Comment:
FWIW the tracerbench stats class exposes a seven-fig-summary already. 
https://github.com/TracerBench/tracerbench/blob/master/packages/stats/src/stats.ts#L41-L44